olp-cpp-sdk-core/include/olp/core/cache/CacheSettings.h

```diff
@@ -113,6 +122,18 @@ struct CORE_API CacheSettings {
    */
   EvictionPolicy eviction_policy = EvictionPolicy::kLeastRecentlyUsed;
 
+  /**
+   * @brief This flag sets the compression policy to be applied on the database.
+   *
+   * Even if the input data is incompressible, the kSnappyCompression
+   * implementation will efficiently detect that and will switch to uncompressed
+   * mode. In some cases though, when the data to be inserted is already
+   * compressed by any means, e.g. protobuf or other serialization protocols, it
+   * might not be worth it to enable any compression at all as it will eat up
+   * some CPU. The default value is CompressionType::kSnappyCompression.
```

Comment:
Should we add information that the parameter is dynamic and can be changed between runs?

Reply:
Will add.